File: cash.py
def main():
    # Get the amount of change owed from the user
    while True:
        try:
            change = float(input("Change owed: "))
            if change > 0:
                break
        except ValueError:
            continue

    # Convert dollars to cents
    cents = round(change * 100)

    # Calculate the minimum number of coins
    coins = 0

    # Quarters
    coins += cents // 25
    cents %= 25

    # Dimes
    coins += cents // 10
    cents %= 10

    # Nickels
    coins += cents // 5
    cents %= 5

    # Pennies
    coins += cents

    # Output the minimum number of coins
    print(coins)


if __name__ == "__main__":
    main()

File: cash.c
#include <cs50.h> // This includes the CS50 library for get_int function
#include <stdio.h>

int main(void)
{
    int cents;

    // Prompt user for the amount of change owed
    do
    {
        cents = get_int("Change owed: ");
    }
    while (cents < 0);

    int coins = 0;

    // Calculate the number of quarters (25 cents)
    coins += cents / 25;
    cents %= 25;

    // Calculate the number of dimes (10 cents)
    coins += cents / 10;
    cents %= 10;

    // Calculate the number of nickels (5 cents)
    coins += cents / 5;
    cents %= 5;

    // Calculate the number of pennies (1 cent)
    coins += cents / 1;

    // Print the total number of coins
    printf("%d\n", coins);
}
